src/platforms/ruby/common/configuration/integration_options.mdx

```diff
@@ -23,3 +23,29 @@ config.rails.report_rescued_exceptions
 ```ruby
 config.rails.skippable_job_adapters = ["ActiveJob::QueueAdapters::MyAdapter"]
 ```
+
+
+`rails.tracing_subscribers`
+
+: Tracing subscribers are responsible for capturing tracing spans from Rails instrumentation. The default subscribers are:
+
+- `Sentry::Rails::Tracing::ActionControllerSubscriber`
+- `Sentry::Rails::Tracing::ActionViewSubscriber`
+- `Sentry::Rails::Tracing::ActiveRecordSubscriber`
+
+```ruby
+# you can add a new subscriber
+config.rails.tracing_subscribers << MySubscriber
+# or replace the set completely
+config.rails.tracing_subscribers = [MySubscriber]
+```
+
+## sentry-delayed_job
+
+`delayed_job.report_after_job_retries`
+
+: `sentry-delayed_job` captures exceptions whenever they occur in a job execution. You can make it wait utill the retry is exhausted. Default is `false`.
```

Comment:
```suggestion
: `sentry-delayed_job` captures exceptions whenever they occur in a job execution. You can make it wait until the retry is exhausted. Default is `false`.
```

---

Is it really `sentry-delayed_job` with `-` and `_` in the same word?

Reply:
yes. `delayed_job` is the gem's name and I don't want to change it.